c_src/quicer_config.c

```diff
@@ -335,6 +335,13 @@ getopt3(ErlNifEnv *env, __unused_parm__ int argc,
     }
   else if (IS_SAME_TERM(eopt, ATOM_QUIC_PARAM_CONN_STATISTICS))
     {
+      if (q_ctx && Level == QUIC_PARAM_LEVEL_STREAM)
```

Comment:
what variables start with uppercase and what do not?

Reply:
There is no standard for the moment. 

I try to keep naming to follow like:

if it is borrowed from or there is a reflection in msquic, we use the same Var name.

if it is pure quicer internal var, snake name is preferred.

Nif sits in the middle of erlang code and msquic, the styling becomes a problem indeeded.

Feel free to change if something is undesired/ugly.